base_gs1_barcode/__manifest__.py

```diff
@@ -1,70 +1,49 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    This module is copyright (C) 2012 Numérigraphe SARL. All Rights Reserved.
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU General Public License as published by
-#    the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU General Public License for more details.
-#
-#    You should have received a copy of the GNU General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2012 Numérigraphe SARL. All Rights Reserved.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Decoding API for GS1-128 (aka UCC/EAN-128) and GS1-Datamatrix',
-    'version': '1.0',
+    'version': '10.0.1.0.0',
     'author': u'Numérigraphe',
     'website': 'http://numerigraphe.com',
     'category': 'Generic Modules/Inventory Control',
     'description': """
-This module provides an API to decoding the content of structured barcodes \
+This module provides an API to decoding the content of structured barcodes
```

Comment:
could you move all the description into the readme file ?

Reply:
yes please